[2025-10-26 13:05:25,374] [INFO] [real_accelerator.py:191:get_accelerator] Setting ds_accelerator to cuda (auto detect)
usage: run.py [-h] --model_type MODEL_TYPE --model_name_or_path
              MODEL_NAME_OR_PATH --output_dir OUTPUT_DIR
              [--load_model_path LOAD_MODEL_PATH]
              [--prune_strategy {leancode,dietcode,leancode_d,None,slimcode}]
              [--train_filename TRAIN_FILENAME] [--dev_filename DEV_FILENAME]
              [--test_filename TEST_FILENAME] [--config_name CONFIG_NAME]
              [--tokenizer_name TOKENIZER_NAME]
              [--max_source_length MAX_SOURCE_LENGTH]
              [--max_target_length MAX_TARGET_LENGTH] [--do_train] [--do_eval]
              [--do_test] [--do_lower_case] [--no_cuda]
              [--train_batch_size TRAIN_BATCH_SIZE]
              [--eval_batch_size EVAL_BATCH_SIZE]
              [--gradient_accumulation_steps GRADIENT_ACCUMULATION_STEPS]
              [--learning_rate LEARNING_RATE] [--beam_size BEAM_SIZE]
              [--weight_decay WEIGHT_DECAY] [--adam_epsilon ADAM_EPSILON]
              [--max_grad_norm MAX_GRAD_NORM] [--max_steps MAX_STEPS]
              [--eval_steps EVAL_STEPS] [--train_steps TRAIN_STEPS]
              [--warmup_steps WARMUP_STEPS] [--local_rank LOCAL_RANK]
              [--seed SEED] [--lang LANG] [--gen_weight]
              [--num_train_epochs NUM_TRAIN_EPOCHS] [--ratio RATIO]
run.py: error: the following arguments are required: --model_type, --model_name_or_path, --output_dir
